src/lib/config.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
   PORT: RC.PORT,
   // ADMIN_PORT: RC.ADMIN_PORT,
   DATABASE_URI: RC.DATABASE_URI,
+  POOL_MIN: RC.DB_CONNECTION.POOL_MIN,
+  POOL_MAX: RC.DB_CONNECTION.POOL_MAX,
```

Comment:
Why was this changed? This was fine. `DB_CONNECTION_POOL_MAX`?  It has no context now.

Reply:
Revert updates.